src/test/java/teammates/test/cases/browsertests/InstructorCourseDetailsPageUiTest.java

```diff
@@ -239,14 +237,14 @@ private InstructorCourseDetailsPage getCourseDetailsPage() {
         return loginAdminToPage(detailsPageUrl, InstructorCourseDetailsPage.class);
     }
 
-    private boolean didStudentReceiveReminder(String courseName, String courseId, String studentEmail,
-                                              String studentPassword)
+    // TODO replace with GAE LocalMailService stub as we are checking the logic here, not reliability of email delivery
```

Comment:
The LocalMailService stub should only be used when email is not enabled (i.e. in the dev server):

```
if (isEmailEnabled) {
    ...
} else {
    // TODO: use GAE LocalMailService
}
```

For staging server testing, we really are testing email delivery, as it is a real world test.